blog/external-css.ftd

```diff
@@ -29,51 +22,59 @@ published-on: November 3, 2023 at 10:27 am
 post-url: /external-css/
 author: $authors.rithik
 
-In the ever-evolving world of web development, staying up-to-date with
-the latest CSS (Cascading Style Sheets) properties is crucial to make the most
-out of it. These CSS properties keeps on changing with time
-while certain properties become deprecated, newer, more advanced ones emerge.
-Some of these emerging properties are released in experimental stages, while
-some of them could be very newly released raising questions about their
-compatibility with fastn. In such scenarios, the importance of external CSS
-becomes evident. In this blog, we will see how we can make use of external CSS
-in fastn.
+In this blog, we will break down all the necessary details we
+need to know before using external CSS properties on our fastn components.
```

Comment:
Why `index-meta` and `meta` both used?

Reply:
That were initial values, now those values are different